query-service/src/integrationTest/java/org/hypertrace/core/query/service/QueryServiceTestUtils.java

```diff
@@ -31,4 +32,61 @@ public static Filter createFilter(
     Expression rhs = Expression.newBuilder().setLiteral(constant).build();
     return Filter.newBuilder().setLhs(lhs).setOperator(op).setRhs(rhs).build();
   }
+
+  public static Function.Builder createTimeColumnGroupByFunction(
```

Comment:
Looks like this same function `createTimeColumnGroupByFunction` is copied twice.

Reply:
One takes long and another takes string argument.